### PR TITLE
Button component

### PR DESCRIPTION
**Pull request by KoushikKintali**
Button Component:

You can view the UI

https://nan-ui.netlify.app/components/button/button.html

### CODE REVIEW

**Review by msdeshmukh009 — APPROVED** [review]
I think the styles related to filled buttons are not present in `button.css`. It is better if you can put all the button styles in one place. Also, add hover to filled buttons it will look good.

**Review by manojkumar8400 — APPROVED** [review]
looks good to me.

**Comment by preethasri** [comment]
Looks good

**Comment by jyothishp994** [comment]
Nan is a beautiful component library. Found an issue where sidebar that gets converted to a dropdown not listing down on smaller screensizes. I think giving flex-direction to column on doc-container on smaller screensizes is simplest way to handle responsiveness.

File: index.css [hunk 1]
```diff
@@ -175,7 +176,7 @@ h4 {
     border: thin solid var(--color-primary);
     background-color: var(--color-primary);
     color: var(--color-white);
-}
```
Comment (jyothishp994) [inline index.css:178]: Added var for colors, That is good

File: components/button/button.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+                    src="https://carbon.now.sh/embed/1CF8EUR30q52DyQrNXRh"
+                    style="width: 100%; height: 202px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                  </iframe>
+                </div>
+            </section>
+        </div>
+    </main>
+</body>
+</html>
```
Comment (jyothishp994) [inline components/button/button.html:168]: Very well indented code

**Review by jyothishp994 — COMMENTED** [review]
Code is clean and well indented. Made use of Var for colors, Used relative size than px. Good to merge

**Pull request merged**